tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -13,8 +13,8 @@
  */
 namespace Cake\Test\TestCase\Network\Http\Adapter;
 
-use Cake\Network\Http\Adapter\Stream;
-use Cake\Network\Http\Request;
+use Cake\Http\Client\Adapter\Stream;
+use Cake\Http\Client\Request;
```

Comment:
I had originally left these alone to ensure backwards compatibility with earlier 3.x was retained.